source/class/qx/data/controller/MSelection.js

```diff
@@ -61,6 +61,22 @@ qx.Mixin.define("qx.data.controller.MSelection", {
       event: "changeSelection",
       apply: "_applySelection",
       init: null
+    },
+
+    /**
+     * If set to true, the selection property will be allowed to have items which are not in the model property.
+     * If the model property changes, this will not cause the selection property to be reset,
+     * If the selection contains items not in the model,
+     * the target widget will only select the items that are in the model.
```

Comment:
Can you elaborate a bit, I don't understand what will happen here with the 'non-model' item.